### ISSUE TITLE
Tons of warnings just by importing tf (2.0.0-beta1)

### ISSUE THREAD

**Issue by durandg12** [tensorflow/tensorflow#31364]
**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow): 
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): macOS 10.13.6
- TensorFlow installed from (source or binary): pip install tensorflow==2.0.0-beta1
- TensorFlow version (use command below): v2.0.0-beta0-16-g1d91213fe7 2.0.0-beta1
- Python version: v3.6.7:6ec5cf24b7, Oct 20 2018, 03:02:14

**Describe the current behavior**

I just created a virtual environnement to test tensorflow 2.0.0-beta1 with no other package installed. Just calling `import tensorflow` launched many warning messages. It is strange that I already use 2.0.0-beta1 in other virtual environnements and never saw these warnings before.

**Code to reproduce the issue**

In bash:
```
python3 -m venv testenv
source testenv/bin/activate
pip install tensorflow==2.0.0-beta1
python
```
Then in python:
```
import tensorflow as tf
# print(tf.version.GIT_VERSION, tf.version.VERSION)
 ```

**Warning messages**
```
/path-to-testenv/testenv/lib/python3.6/site-packages/tensorflow/python/framework/dtypes.py:516: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint8 = np.dtype([("qint8", np.int8, 1)])
/path-to-testenv/testenv/lib/python3.6/site-packages/tensorflow/python/framework/dtypes.py:517: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint8 = np.dtype([("quint8", np.uint8, 1)])
/path-to-testenv/testenv/lib/python3.6/site-packages/tensorflow/python/framework/dtypes.py:518: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint16 = np.dtype([("qint16", np.int16, 1)])
/path-to-testenv/testenv/lib/python3.6/site-packages/tensorflow/python/framework/dtypes.py:519: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint16 = np.dtype([("quint16", np.uint16, 1)])
/path-to-testenv/testenv/lib/python3.6/site-packages/tensorflow/python/framework/dtypes.py:520: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint32 = np.dtype([("qint32", np.int32, 1)])
/path-to-testenv/testenv/lib/python3.6/site-packages/tensorflow/python/framework/dtypes.py:525: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  np_resource = np.dtype([("resource", np.ubyte, 1)])
/path-to-testenv/testenv/lib/python3.6/site-packages/tensorboard/compat/tensorflow_stub/dtypes.py:541: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint8 = np.dtype([("qint8", np.int8, 1)])
/path-to-testenv/testenv/lib/python3.6/site-packages/tensorboard/compat/tensorflow_stub/dtypes.py:542: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint8 = np.dtype([("quint8", np.uint8, 1)])
/path-to-testenv/testenv/lib/python3.6/site-packages/tensorboard/compat/tensorflow_stub/dtypes.py:543: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint16 = np.dtype([("qint16", np.int16, 1)])
/path-to-testenv/testenv/lib/python3.6/site-packages/tensorboard/compat/tensorflow_stub/dtypes.py:544: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint16 = np.dtype([("quint16", np.uint16, 1)])
/path-to-testenv/testenv/lib/python3.6/site-packages/tensorboard/compat/tensorflow_stub/dtypes.py:545: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint32 = np.dtype([("qint32", np.int32, 1)])
/path-to-testenv/testenv/lib/python3.6/site-packages/tensorboard/compat/tensorflow_stub/dtypes.py:550: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  np_resource = np.dtype([("resource", np.ubyte, 1)])
```

**Comment by ravikyram** [comment]
@durandg12 

Can you please let us know which numpy version you are using. Thanks!

**Comment by durandg12** [comment]
@ravikyram I do not install numpy myself in the virtual environnement. However, when I install tensorflow 2.0.0-beta1 it installs numpy 1.17.0.

```
$pip install tensorflow==2.0.0-beta1
...
Collecting numpy<2.0,>=1.14.5 (from tensorflow==2.0.0-beta1)
  Using cached https://files.pythonhosted.org/packages/be/e8/45079ae05c4dda4a67bc51578ae5e75feda0a79c2836d477d676e7a58efb/numpy-1.17.0-cp36-cp36m-macosx_10_6_intel.macosx_10_9_intel.macosx_10_9_x86_64.macosx_10_10_intel.macosx_10_10_x86_64.whl
...
```

```
$ pip freeze
absl-py==0.7.1
astor==0.8.0
gast==0.2.2
google-pasta==0.1.7
grpcio==1.22.0
h5py==2.9.0
Keras-Applications==1.0.8
Keras-Preprocessing==1.1.0
Markdown==3.1.1
numpy==1.17.0
protobuf==3.9.1
six==1.12.0
tb-nightly==1.14.0a20190603
tensorflow==2.0.0b1
termcolor==1.1.0
tf-estimator-nightly==1.14.0.dev2019060501
Werkzeug==0.15.5
wrapt==1.11.2
```

**Comment by ravikyram** [comment]
I think TF 2.0.b1 comes with numpy dependency 1.16.4 and it shows no warning. 

However, numpy 1.17.0 ,tensorflow could be updated to make it compatible with numpy 1.17. Created a PR #30559 for the fix.
Also, please refer #30427 for more insights.Thanks!

**Comment by durandg12** [comment]
From what I understand, TF 2.0.b1 is designed to work with numpy 1.16.4, but it installs 1.17. Isn't that a little illogical ? Anyway, since you already created a PR that has been merged, I assume that the warnings will disappear in the near future and that we can close this issue.

**Comment by mihaimaruseac** [comment]
See also https://github.com/tensorflow/tensorflow/issues/31249 and https://github.com/tensorflow/tensorflow/pull/30559

TL;DR: Both the RC0 for TF2.0 and the patch 1.14.1 will have these warnings solved already.

**Comment by durandg12** [comment]
What does RC0 mean ?

**Comment by mihaimaruseac** [comment]
release candidate 0, the next step after the betas.

There will be a few RCs and then 2.0 will be officially launched in a final form

**Comment by durandg12** [comment]
Thank you. I can close this issue then.

**Comment by kumarnikhil936** [comment]
You can simply revert back to the numpy 1.16.1 version, by 
→ pip install numpy==1.16.1

**Comment by durandg12** [comment]
> Or you can test with the newest released RC0 for TF 2.0
> 
> We recommend testing the release candidates as after the final release we don't do patch releases except for security related issues

@mihaimaruseac I did it and the warnings are gone, thanks! So this issue can stay closed.
Strangely, the warnings from [this issue](https://github.com/tensorflow/tensorflow/issues/29881) are not gone whereas the issue has been closed 2 months ago by [the following commit](https://github.com/tensorflow/tensorflow/commit/d7e858192d1de827bc03705ac62e1bd38daf06d8).
